pages/common/yank.md

```diff
@@ -0,0 +1,23 @@
+# yank
+
+> Read input from stdin and display a selection interface that allows a field to be selected and copied to the clipboard.
+
+- Yank an environment variable key or value:
+
+`env | yank -d =`
+
+- Yank a field from a CSV file:
+
+`yank -d \", <file.csv`
+
+- Yank a whole line using the -l option:
+
+`make 2>&1 | yank -l`
```

Comment:
Change this to - `echo -e "hello\nworld" | yank -l`

Reply:
I chose `echo` because it's a simple command and shows the functionality clearly that it is printing multiple lines. But if you think `dmesg` is better, that is also fine.